### PR TITLE
Add mention of `is_a()` and `instanceof` in `class_implements()` and `class_parents()` pages

### PR DESCRIPTION
**Pull request by alexandre-daubois**
Fix https://github.com/php/doc-en/issues/1606

### CODE REVIEW

File: reference/spl/functions/class-implements.xml [hunk 1]
```diff
@@ -99,12 +99,24 @@ Array
   </para>
  </refsect1>
 
+ <refsect1 role="notes">
+  &reftitle.notes;
+  &note.no-remote;
```
Comment (Girgias) [inline reference/spl/functions/class-implements.xml:104]: Why this entity?
Doesn't seem related at all and something from a copy/paste.

File: reference/spl/functions/class-implements.xml [hunk 2]
```diff
@@ -99,12 +99,24 @@ Array
...
+  <note>
+   <simpara>
+    To check that an object implements an interface, you should either use &instanceof;
+    or the <function>is_a</function> function.
```
Comment (Girgias) [inline reference/spl/functions/class-implements.xml:108]: Personalisation:
```suggestion
    To check that an object implements an interface, &instanceof;
    or the <function>is_a</function> function should be used instead.
```

**Review by Girgias — CHANGES_REQUESTED** [review]

File: reference/spl/functions/class-implements.xml [hunk 3]
```diff
@@ -99,12 +99,24 @@ Array
...
+  </note>
+ </refsect1>
+
  <refsect1 role="seealso">
   &reftitle.seealso;
   <para>
    <simplelist>
     <member><function>class_parents</function></member>
     <member><function>get_declared_interfaces</function></member>
+    <member><function>is_a</function></member>
```
Comment (Girgias) [inline reference/spl/functions/class-implements.xml:119]: Instanceof should also be mentionned here

Comment (alexandre-daubois) [reply reference/spl/functions/class-implements.xml:118]: Added :slightly_smiling_face:

File: reference/spl/functions/class-implements.xml (hunk 2 above)
Comment (alexandre-daubois) [reply reference/spl/functions/class-implements.xml]: Updated!

**Review by Girgias — APPROVED** [review]

**Pull request merged**